x-pack/test/reporting/api/bwc_existing_indexes.js

```diff
@@ -28,14 +28,16 @@ export default function ({ getService }) {
     describe('existing 6_2 index', () => {
       before('load data and add index alias', async () => {
         await reportingAPI.deleteAllReportingIndexes();
-        await esArchiver.load('bwc/6_2');
+        await esArchiver.load('reporting/bwc/6_2');
```

Comment:
@tsullivan hey sir!  Good idea on this pr imho.
One thing though, this test is `.skip()`'d, maybe unskip and test since we are refactoring?
Looks like it was skipped in September.

Reply:
The BWC tests will be removed in an upcoming PR, since Reporting will no longer be backwards compatible with the legacy 6.x generation URLs as of 8.0. So I don't think we're really interested in seeing that it will start to pass.